test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -219,3 +240,83 @@
                     :y "y"
                     :z "y"}
                   (response->status resp)))))))))
+
+(deftest proxy-ssl-status-endpoint-test
```

Comment:
I played around with running these tests a bit, and this particular test isn't being run. It's also not being registered as a test (if I comment it out or remove it, the number of tests/assertions run by leiningen doesn't decrease when I run `lein test`). 

Moving it to the bottom of the file causes the assertions to be run, but again, if I comment out the test the number of tests/assertions run by leiningen does not decrease. I have no idea why this is happening but it seems really weird.